doc/jvm/xgboost4j_spark_gpu_tutorial.rst

```diff
@@ -211,32 +215,31 @@ and the prediction for each instance.
 Submit the application
 **********************
 
-Here’s an example to submit an end-to-end XGBoost-4j-Spark-GPU Spark application to an
-Apache Spark Standalone cluster, assuming the application main class is Iris and the
-application jar is iris-1.0.0.jar
+Assuming that the application main class is "Iris" and the application jar is "iris-1.0.0.jar,"
+provided below is an instance demonstrating how to submit the xgboost application to the Apache
+Spark Standalone cluster.
 
 .. code-block:: bash
 
-  cudf_version=22.02.0
-  rapids_version=22.02.0
-  xgboost_version=1.6.1
+  rapids_version=23.10.0
+  xgboost_version=2.0.1
   main_class=Iris
   app_jar=iris-1.0.0.jar
 
   spark-submit \
     --master $master \
-    --packages ai.rapids:cudf:${cudf_version},com.nvidia:rapids-4-spark_2.12:${rapids_version},ml.dmlc:xgboost4j-gpu_2.12:${xgboost_version},ml.dmlc:xgboost4j-spark-gpu_2.12:${xgboost_version} \
+    --packages com.nvidia:rapids-4-spark_2.12:${rapids_version},ml.dmlc:xgboost4j-gpu_2.12:${xgboost_version},ml.dmlc:xgboost4j-spark-gpu_2.12:${xgboost_version} \
     --conf spark.executor.cores=12 \
-    --conf spark.task.cpus=1 \
+    --conf spark.task.cpus=12 \
```

Comment:
why the change?

Reply:
Yeah. Spark can't 100% ensure each training task will grab different GPU. I will add the stage-level scheduling in JVM package to fix that.